groups/bdl/bdlb/bdlb_guid.t.cpp

```diff
@@ -407,6 +409,46 @@ int main(int argc, char *argv[])
         }
 //..
       } break;
+      case 15: {
+        // --------------------------------------------------------------------
+        // TESTING HASH FUNCTION
+        //
+        // Concerns:
+        //: 1 A guid object can be hashed by instances of the 'bslh::Hash'
+        //:   template and 'bsl::hash<bdlb::Guid>'.
+        //:
+        //: 2 A small sample of different guid objects produce different
+        //:   hashes.
+        //
+        // Plan:
+        //: 1 Hash nine different guid objects and verify 'bslh::Hash<>'
+        //:   produces the same result as 'bsl::hash<bdlb::Guid>'
+        //:
+        //: 2 Hash nine different guid objects and verify they produce
+        //:   nine different hashes.
+        //
+        // Testing:
+        //   bsl::hash<bdlb::Guid>
+        //   bslh::Hash<>
+        // --------------------------------------------------------------------
+        if (verbose) cout << endl
+                          << "TESTING HASH FUNCTION" << endl
+                          << "=====================" << endl;
+
+        const bsl::size_t NUM_VALUES = sizeof(VALUES) / sizeof(VALUES[0]);
+
+        bsl::hash<Obj> bsl_hash_function;
```

Comment:
Coding standards naming convention for variables is camel case (`bslHashFunction`, `bslhHashFunction`, `hashResults` etc)